frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -106,6 +106,50 @@ if Device:setDateTime() then
     }
 end
 
+if Device:isCervantes() or Device:isKobo() or Device:isSDL() or Device:isSonyPRSTUX() then
+    common_settings.autosuspend = {
```

Comment:
If this whole thing comes from the plugin doesn't it make more a lot more sense to simply stick it in the plugin? (MenuSorter doesn't care where it came from.)

Reply:
I wasn't aware we could do that, but, yeah, that makes sense. Scratched my head a bit because AutoSuspend was layed out a bit weirdly, but finally figured it out ;).

Reply:
One of the main reasons I wrote it. ;-)